src/zope/app/apidoc/utilities.py

```diff
@@ -325,10 +366,10 @@ def renderText(text, module=None, format=None, dedent=True):
 
     assert format in _format_dict.values()
 
-    text = dedentString(text)
-
-    if not isinstance(text, unicode):
+    if isinstance(text, bytes):
         text = text.decode('latin-1', 'replace')
```

Comment:
(Also, I must say I've been reviewing this rather shallowly without diving deep into the context -- e.g. where do these byte strings come from?  I just applying general principles: I want Latin-1 to die. ;)